contracts/.gitignore

```diff
@@ -0,0 +1,14 @@
+# Compiler files
+cache/
+out/
+
+# Ignores development broadcast logs
+!/broadcast
+/broadcast/*/31337/
+/broadcast/**/dry-run/
+
+# Docs
+docs/
```

Comment:
Why is docs ignored?

Reply:
Its just the foundry defaults, not clear why.  Removed.